app/helpers/job_application_helper.rb

```diff
@@ -16,4 +16,12 @@ def publisher_job_application_status_tag(status)
               colour: JOB_APPLICATION_STATUS_TAG_COLOURS[status.to_sym],
               classes: "govuk-!-margin-bottom-2"
   end
+
+  def status_tag_colour(status)
+    JOB_APPLICATION_STATUS_TAG_COLOURS[status.to_sym]
+  end
+
+  def deadline_passed?(date)
```

Comment:
Does the `expired` scope on Vacancy not meet your needs? (That returns _published_ vacancies that have a past expiry_date)

Reply:
not too sure what you mean here, i dont see how expired scope helps for what this is needed for. ill pick this up with you tues

Reply:
I'm not sure what the need for this method is either?